src/main/java/com/artipie/rpm/http/RpmUpload.java

```diff
@@ -146,12 +147,16 @@ public Key file() {
          * @return Override param value, <code>false</code> - if absent
          */
         public boolean override() {
-            return Optional.ofNullable(new RequestLineFrom(this.line).uri().getQuery())
-                .map(
-                    query -> Streams.stream(Splitter.on("&").split(query))
-                        .anyMatch(part -> part.equals("override=true"))
-                )
-                .orElse(false);
+            return this.params().anyMatch(part -> part.equals("override=true"));
```

Comment:
@olegmoz `part -> part.equals("...")` can be extracted to a private method to make code more readable and avoid code duplication. We can have than `this.params().anyMatch(equalTo("override=true")) `

Reply:
@Vatavuk agreed, please review the update